test/parallel/test-http-pause.js

```diff
@@ -47,9 +47,9 @@ const server = http.createServer((req, res) => {
   }, 100);
 });
 
-server.listen(0, function() {
+server.listen(0, () => {
```

Comment:
```suggestion
server.listen(0, function() {
  // Anonymous function rather than arrow function to test `this` value.
  assert.strictEqual(this, server);
```

Reply:
Instead of a comment to explain why the function is anonymous, another option is to name the function (for example `function handleListening() { ... }`)